LICENSE

```diff
@@ -1,4 +1,6 @@
 Copyright (c) 2015, Ross Kinder
```

Comment:
Can we say 2015-2024 (or whatever is correct) here?

Reply:
Done!